### PR TITLE
Add a few non-goals

### PR DESCRIPTION
**Pull request by ayuishii**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -106,9 +106,11 @@ for specifying the policies for each individual API.
 
 ## Non-goals
 
-[If there are “adjacent” goals which may appear to be in scope but aren’t,
-enumerate them here. This section may be fleshed out as your design
-progresses and you encounter necessary technical and other trade-offs.]
+* Sharding for cookies
```
Comment (pwnall) [inline explainer.md:109]: * Integration with storage mechanisms that don't follow the same origin policy, such as cookies.

File: explainer.md [hunk 2]
```diff
@@ -106,9 +106,11 @@ for specifying the policies for each individual API.
...
+
+* Change behavior for third party context
```
Comment (pwnall) [inline explainer.md:111]: * Behavior in third-party contexts is deferred to future work, to keep this explainer manageable.

File: explainer.md [hunk 3]
```diff
@@ -106,9 +106,11 @@ for specifying the policies for each individual API.
...
+
+* Standardizing browser eviction
```
Comment (pwnall) [inline explainer.md:113]: * Standardizing user agent behavior around storage eviction

**Review by pwnall — APPROVED** [review]
This is a good first cut, thank you!

LGTM w/ minor revisions applied.

**Comment by ayuishii** [comment]
Thanks for the reveiw!

**Pull request merged**